templates/index.html

```diff
@@ -168,6 +174,10 @@ <h3><u>Contact Us</u></h3>
         document.getElementById("sign-up-form").hidden = false;
       }
 
+      function pray() {
```

Comment:
ahh sorry, here on the frontend you should leave this javascript function as `clickPrayerButton`, but the backend url resource should be `pray` as you have changed it